deluge/conftest.py

```diff
@@ -0,0 +1,158 @@
+#
```

Comment:
Not wild about this file not being in the `tests` subfolder, but I didn't know how to make it apply to plugin tests as well without moving it up to the root.

Reply:
This should be excluded from the packaging if it's in this dir. 

Is this [pytest issue](https://github.com/pytest-dev/pytest/issues/5822#issuecomment-697331920) the problem? Can we try the suggested solution?

Reply:
> The simplest solution is to just move the conftest.py up one directory (repo1/conftest.py).

This suggested solution is the one I went with.

Just tried the second solution of putting a conftest in both of our plugins that have tests test folders, looks like that suggested method isn't supported anymore.

```
Defining 'pytest_plugins' in a non-top-level conftest is no longer supported:
It affects the entire test suite instead of just below the conftest as expected.
  C:\Users\chase\PycharmProjects\deluge\deluge\plugins\Stats\deluge_stats\tests\conftest.py
Please move it to a top level conftest file at the rootdir:
  C:\Users\chase\PycharmProjects\deluge
For more information, visit:
  https://docs.pytest.org/en/latest/deprecations.html#pytest-plugins-in-non-top-level-conftest-files
collected 0 items / 1 error
```

Reply:
To exclude this from the package... that'd be MANIFEST.in?

Reply:
Actually ignore my muttering about excluding it, must have been thinking of something else